local-tests/test.ts

```diff
@@ -0,0 +1,449 @@
+// This test requires using "bun" to run.
+// Installation: https://bun.sh/docs/installation
+// Test command: yarn test:local --filter={testName} --network={network | 'localhost' is default}
+
+import {
+  LitActionResource,
+  LitPKPResource,
+  createSiweMessage,
+  craftAuthSig,
+  AuthCallbackFields,
+  WithRecapFields,
+  CreateSiweType,
+} from '@lit-protocol/auth-helpers';
+import { devEnv } from './setup/env-setup';
+import * as ethers from 'ethers';
+import { getNetworkFlag, showTests, runTests } from './setup/utils';
+import { AuthCallbackParams, LitAbility } from '@lit-protocol/types';
+import { AUTHSIG_ALGO } from '@lit-protocol/constants';
+
+const devEnvPromise = devEnv({
+  env: getNetworkFlag(),
+  debug: true,
+});
+
+const tests = {
+  /**
+   * Test Commands:
+   * ✅ yarn test:local --filter=testUseAuthSigToExecuteJsConsoleLog --network=habanero --version=v0
+   * ✅ yarn test:local --filter=testUseAuthSigToExecuteJsConsoleLog --network=localchain --version=v0
+   */
+  testUseAuthSigToExecuteJsConsoleLog: async () => {
+    const res = await litNodeClient.executeJs({
+      authSig: hotWalletAuthSig,
+      code: `(async() => {
+        console.log("Testing!");
+      })()`,
+      jsParams: {},
+    });
+
+    if (res && res.success) {
+      console.log('✅ res:', res);
+    }
+  },
+
+  /**
+   * Test Commands:
+   * ✅ yarn test:local --filter=testUseAuthSigToExecuteJsSigning --network=habanero --version=v0
+   * ✅ yarn test:local --filter=testUseAuthSigToExecuteJsSigning --network=localchain --version=v0
+   */
+  testUseAuthSigToExecuteJsSigning: async () => {
+    const TO_SIGN = ethers.utils.arrayify(
+      ethers.utils.keccak256([1, 2, 3, 4, 5])
+    );
+
+    try {
+      const res = await litNodeClient.executeJs({
+        authSig: hotWalletAuthSig,
+        code: `(async () => {
+          const sigShare = await LitActions.signEcdsa({
+            toSign: dataToSign,
+            publicKey,
+            sigName: "sig",
+          });
+        })();`,
+        jsParams: {
+          dataToSign: TO_SIGN,
+          publicKey: hotWalletOwnedPkp.publicKey,
+        },
+      });
+
+      if (res.signatures.sig.signature) {
+        console.log('✅ res:', res);
+      } else {
+        console.log('❌ Error res:', res);
+      }
+    } catch (e) {
+      console.log('Error:', e);
+    }
+  },
+
+  /**
+   * Test Commands:
+   * ✅ yarn test:local --filter=testGetSessionSigs --network=habanero --version=v0
+   * ✅ yarn test:local --filter=testGetSessionSigs --network=localchain --version=v1
+   */
+  testGetSessionSigs: async () => {
```

Comment:
Maybe rename it `getEoaSessionSigs`?

Reply:
If we are not testing the functionality of getting sessionSigs alone, I think we can just remove the test

Reply:
This is completely SDK side right? So no need to check this?